lib/buildInEnums.js

```diff
@@ -0,0 +1,167 @@
+export const predefinedActions = {
+  androidCopyImages: 'android/copyImages',
+  copyAssets: 'copy_assets',
+};
+
+export const logWarningLevels = {
+  warn: 'warn',
+  error: 'error',
+  disabled: 'disabled',
+};
+
+export const logVerbosityLevels = {
+  default: 'default',
+  silent: 'silent',
+  verbose: 'verbose',
+};
+
+export const logBrokenReferenceLevels = {
+  throw: 'throw',
+  console: 'console',
+};
+
+export const commentStyles = {
+  short: 'short',
+  long: 'long',
+  none: 'none',
+};
+
+export const commentPositions = {
+  above: 'above',
+  inline: 'inline',
+};
+
+export const commentStylesFileHeader = {
```

Comment:
```suggestion
export const commentStylesFileHeaders = {
```

Reply:
After your suggestion, I thougth about it. Maybe the most precise and best understandable name would be `fileHeaderCommentStyles`, when I think about it!?

Reply:
Yea that's better indeed